src/index.ts

```diff
@@ -1,14 +1,49 @@
 import * as fs from 'fs';
 import * as path from 'path';
+import * as loadPostCssConfig from 'postcss-load-config';
 import * as ts_module from 'typescript/lib/tsserverlibrary';
+
 import { createMatchers } from './helpers/createMatchers';
 import { isCSSFn } from './helpers/cssExtensions';
 import { getDtsSnapshot } from './helpers/cssSnapshots';
 import { Options } from './options';
 
+import * as postcss from 'postcss';
+import * as postcssIcssSelectors from 'postcss-icss-selectors';
+
+const removePlugin = postcss.plugin('remove-mixins', () => (css) => {
+  css.walkRules((rule) => {
+    rule.walkAtRules((atRule) => {
+      if (atRule.name === 'mixin') {
+        atRule.remove();
+      }
+    });
+  });
+});
+
+function getPostCssConfig(dir: string) {
+  try {
+    return loadPostCssConfig.sync({}, dir);
+  } catch (error) {
+    return { plugins: [] };
+  }
+}
+
 function init({ typescript: ts }: { typescript: typeof ts_module }) {
   let _isCSS: isCSSFn;
+
   function create(info: ts.server.PluginCreateInfo) {
+    const postcssConfig = getPostCssConfig(info.project.getCurrentDirectory());
```

Comment:
This tool looks great, but I see it's not _official_ - so I should probably add that to the docs. What do you think?

Reply:
I'm not sure that's true. You can see that they use this package in `postcss-cli` here https://github.com/postcss/postcss-cli/blob/master/index.js#L14